Contributing.md

```diff
@@ -30,6 +36,60 @@ Contributor Agreement (ECA) on file.
 For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
```

Comment:
For ECA, we could additionally reference this guide, which gives more details and is more helpful than the Eclipse resources: https://github.com/adoptium/documentation/blob/main/documentation-documentation/eca-sign-off.adoc